src/components/Counter/Counter.js

```diff
@@ -0,0 +1,41 @@
+import { Button, TextField , Typography} from "@mui/material";
+import React, { useState } from "react";
+import "./counter.css";
+
+const Counter = () => {
+  const [count, setCount] = useState(0);
+
+  return (
+    <div className="root">
+      <Typography variant="h4"  >Counter App</ Typography>
```

Comment:
this should go in app as it is not needed in every component

Reply:
okay